pkg/controller/node/nodecontroller.go

```diff
@@ -215,7 +215,7 @@ func NewNodeController(
 	runTaintManager bool,
 	useTaintBasedEvictions bool) (*NodeController, error) {
 	eventBroadcaster := record.NewBroadcaster()
-	recorder := eventBroadcaster.NewRecorder(api.Scheme, clientv1.EventSource{Component: "controllermanager"})
+	recorder := eventBroadcaster.NewRecorder(api.Scheme, clientv1.EventSource{Component: "node-controller"})
```

Comment:
Yeah, we use `controllermanager` in all places I know of. This change kind of makes sense, but only if we change it everywhere.